ts/BUILD.typescript

```diff
@@ -10,7 +10,7 @@ npm_package_internal(
     name = "npm_typescript",
```

Comment:
does this line need a replacement as well?

Reply:
not necessary since this target represents the npm package.